src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -333,7 +335,7 @@ public static void WriteLongRunningOperationMethod(CodeWriter writer, LowLevelCl
                         writer
                             .Line($"using {typeof(HttpMessage)} {messageVariable:D} = {RequestWriterHelpers.CreateRequestMethodName(startMethod.Name)}({startMethod.Parameters.GetIdentifiersFormattable()});")
                             .AppendRaw("return ")
-                            .WriteMethodCall(async, LroProcessMessageMethodAsyncName, LroProcessMessageMethodName, processMessageParameters);
+                            .WriteMethodCall(async, clientMethod.OperationSchemas.ResponseBodySchema != null ? LroProcessMessageMethodAsyncName : LroProcessMessageWithoutContentMethodAsyncName, clientMethod.OperationSchemas.ResponseBodySchema != null ? LroProcessMessageMethodName : LroProcessMessageWithoutContentMethodName, processMessageParameters);
```

Comment:
This change with the below similar change is a [duplicate code](https://refactoring.guru/smells/duplicate-code). I would suggest extract the similar part into a new method and give it a meaningful name.

Actually, `WritePageableLongRunningOperationMethod` is also a [long method](https://refactoring.guru/smells/long-method) smell. I would suggest divide it too.

Reply:
Yes, we need to refine the method 'WritePageableLongRunningOperationMethod' which current is long and have duplicate code compare the normal LRO Operation method 'WriteLongRunningOperationMethod'. We need to refactor this already existing method. But we will do such refactor in MQ, not in this issue.